packages/amplication-client/src/Resource/git/git-provider-icon-map.ts

```diff
@@ -0,0 +1,13 @@
+import { EnumGitProvider } from "../../models";
+import BitbucketLogo from "../../assets/images/bitbucket.svg";
+import GithubLogo from "../../assets/images/github.svg";
+
+export const gitProviderIconMap = {
+  [EnumGitProvider.Github]: "github",
+  [EnumGitProvider.Bitbucket]: "git-sync", // TODO: replace with bitbucket icon (once we have one)
```

Comment:
Do we have a new issue for this?

Reply:
now we do https://github.com/amplication/amplication/issues/5979